content/Arduino Cloud/General Cloud/Supported-Wi-Fi-bands-for-Arduino-devices.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Supported Wi-Fi bands for Arduino devices"
+id: 14144656051740
+---
+
+Arduino boards and most IoT devices can only be used with 2.4 GHz band networks. The 2.4 GHz band offers better range and signal strength through obstacles compared to 5 GHz.
+
+## How to check if your network is 2.4 GHz
+
+Follow these steps to identify the frequency band of your Wi-Fi network.
+
+**Windows:**
+
+1. Open the settings menu in Windows and select the **Network and Internet** section.
+
+1. Under the wireless network settings, click on the **Hardware properties** link.
+
+1. A properties page will open. Here, you can find details about the wireless network, including the network band (GHz).
+
+**macOS:**
+
+1. Connect to the Wi-Fi network.
+
+1. Click on the Wi-Fi icon on the right side of the top navigation bar while holding down the Option/Alt key.
+
+1. Look for the entry starting with **Channel**. This will specify if the network is in the 2.4 GHz or 5 GHz band.
+
+## Creating a 2.4 GHz network with a hotspot
+
+If your network operates on the 5 GHz band, a quick solution is to create a 2.4 GHz network using the hotspot feature on an Android or iPhone device. iPhone hotspots typically prioritize the 2.4 GHz band by default. Android devices offer users the option to manually select their preferred band for hotspot connections.
```

Comment:
> iPhone hotspots typically prioritize the 2.4 GHz band by default.

Tested and confirmed on my iPhone 11 Pro:

![Screenshot 2024-07-04 at 11 43 44](https://github.com/arduino/help-center-content/assets/71350948/0d850d57-5fa3-4ca7-b4b5-22aaf1fbd8e5)

However, it appears that newer Apple devices are 5 Ghz by default:

> Personal Hotspot uses a 5GHz connection by default. On iPhone 12 or later, you can turn on Maximize Compatibility for Personal Hotspot to use a 2.4GHz connection.
>
> Source: [Using Personal Hotspot](https://it-training.apple.com/tutorials/support/sup040/#:~:text=Personal%20Hotspot%20uses%20a%205GHz%20connection%20by%20default.%20On%20iPhone%2012%20or%20later%2C%20you%20can%20turn%20on%20Maximize%20Compatibility%20for%20Personal%20Hotspot%20to%20use%20a%202.4GHz%20connection.)

> If you're using an iPhone 12 model or later, iPad Pro 11-inch (3rd generation or later), iPad Pro 12.9-inch (5th generation or later), or iPad mini (6th generation), turn on Maximize Compatibility.* Then try connecting your devices again to your Personal Hotspot.
>
> Source: [If Personal Hotspot is not working on your iPhone or iPad (Wi-Fi + Cellular)](https://support.apple.com/en-us/119837#:~:text=If%20you%27re%20using%20an%20iPhone%2012%20model%20or%20later%2C%20iPad%20Pro%2011%2Dinch%20(3rd%20generation%20or%20later)%2C%20iPad%20Pro%2012.9%2Dinch%20(5th%20generation%20or%20later)%2C%20or%20iPad%20mini%20(6th%20generation)%2C%20turn%20on%20Maximize%20Compatibility.*%20Then%20try%20connecting%20your%20devices%20again%20to%20your%20Personal%20Hotspot.)

@gorillagripcore please try the following:

* Include the relevant info about the **Maximize Compatibility** setting on iOS devices.
* Include links to these articles for users that need detailed instructions:
   * [Using Personal Hotspot](https://it-training.apple.com/tutorials/support/sup040/)
   * [Share a mobile connection by hotspot or tethering on Android](https://support.google.com/android/answer/9059108)
* Consider grouping the information for iOS and Android into different list elements, or some other structure, to enhance readability.